html/index.html

```diff
@@ -0,0 +1,190 @@
+                Anointed impartation session with spiritfilled worship by
+                anointed ministers of the Gospel; come and receive empowerment
+                of the Holy Spirit!
+              </p>
+            </div>
+            <div class="card flex">
+              <i class="fa-solid fa-person-chalkboard"></i>
+              <h2 class="tag">Teaching</h2>
+              <p class="tag-info">
+                Anointed teaching session with spiritfilled worship by anointed
+                ministers of the Gospel; come and experience the intimacy in
+                worship!
+              </p>
+            </div>
+            <div class="card flex">
+              <i class="fa-solid fa-fire-flame-curved"></i>
+              <h2 class="tag">Baptism</h2>
+              <p class="tag-info">
+                Anointed baptism session with spiritfilled worship by anointed
+                ministers of the Gospel; come and experience of the baptism of
+                Fire!
+              </p>
+            </div>
+          </div>
+          <a href="#" class="see-prog">SEE THE WHOLE PROGRAM</a>
+          <a href="#" class="mobile">Join The Kingdom Comes Global Worship</a>
+        </div>
+      </section>
+      <!--===== Dynamic content with JS start here ===== -->
+      <section class="speakers flex">
+        <div class="caption flex">
+          <h2>Featured Speakers</h2>
+        </div>
+        <div class="feat-speaker flex"></div>
+        <button class="flex" id="show-more">Show More</button>
+      </section>
+      <!--===== Dynamic content with JS end here ===== -->
+
+      <section class="partner" id="Sponsor">
+        <div class="caption flex">
+          <h2>Partners</h2>
+          <div class="featured"></div>
+        </div>
+        <div class="partner-logo flex">
+          <img
+            class="sponsors"
+            src="/images/ssuc-logo.png"
+            alt="Partner logo"
+          />
+          <img class="sponsors" src="/images/ECS-LOGO.png" alt="Partner logo" />
+          <img
+            class="sponsors"
+            src="/images/SSCC-LOGO.png"
+            alt="Partner logo"
+          />
+          <img
+            class="sponsors"
+            src="/images/Winners-Logo.png"
+            alt="Partner logo"
+          />
+        </div>
+      </section>
+    </main>
+    <!-- === footer section start here === -->
+    <footer>
+      <div class="footer flex">
+        <a href="index.html"
+          ><img  class="logo"
+            src="/images/The-Kingdom-T-Logo.png"
+            alt=" The Kingdom Comes Logo"
+        /></a>
+        <p class="footer-info">
+          &copy;2022 The Kingdom Comes Global Worship Coference 2023; South
+          Sudan. All rights reserve!
+        </p>
+      </div>
+      <!-- ==== Go to top button === -->
+    <button onclick="topFunction()" id="myBtn" title="Go to top"><i class="fa-solid fa-arrow-up-long"></i></button>
+    </footer>
```

Comment:
- [ ] As you can use here the footer doesn't exist in the original design for the mobile version but it still in the desktop version so keep it in the desktop version but remove it from the mobile version 👌🏻👌🏻👌🏻

![image](https://user-images.githubusercontent.com/58553711/182895015-3f837eb4-d204-439b-808e-d6afed0994c4.png)